qemu/tests/numa_stress.py

```diff
@@ -138,12 +131,7 @@ def run(test, params, env):
         stress_test.load_stress_tool()
         error_context.context("Get the qemu process memory use status",
                               test.log.info)
-        node_after, memory_after = max_mem_map_node(host_numa_node, qemu_pid)
-        if node_after == most_used_node and memory_after >= memory_used:
-            test.fail("Memory still stick in node %s" % numa_node_malloc)
-        else:
-            most_used_node = node_after
-            memory_used = memory_after
+        most_used_node, _ = max_mem_map_node(host_numa_node, qemu_pid)
```

Comment:
Is there any reason to get the `most_used_node` again after stress test ?   Since the compare step was removed.

Reply:
Is is be used for the next round ?

Reply:
As we've discussed, the idea is to take _always_ the node in which qemu is running (it should be the same as the most used node) to stress it round by round. As from qemu side is not possible to control numa balancing, the value should be taken at the end of every round, in case of any unexpected change.